### ISSUE TITLE
Option to inject delays to simulate several aspects

### ISSUE THREAD

**Issue by salehsedghpour** [EricssonResearch/cloud-native-app-simulator#18]
· Cross-cluster transport latencies (this may be already done with fortio, just include it in the chain semantics)
· Application processing or queueing latencies
· To specify transport latency as a ratio of the application processing latency

**Comment by alekodu** [comment]
I think the cross-cluster latencies should not be simulated by the application benchmarking tool. This should rather be supported with another tool, e.g., Pumba for container-based clusters.

**Comment by salehsedghpour** [comment]
Yes, these are the early ideas that we had. Maybe we could close it for now.
